libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -329,6 +329,12 @@ class AP_AHRS {
     // returns true on success and results are placed in innovations and variances arguments
     bool get_vel_innovations_and_variances_for_source(uint8_t source, Vector3f &innovations, Vector3f &variances) const WARN_IF_UNUSED;
 
+    //index of active source set used
+    uint8_t active_source_set;
```

Comment:
instead of keeping a copy perhaps the get_active_source_set() should just pull directly from EKF3 if it is available.  If EKF3 is not available (e.g. not compiled in) then it could just return 0.

Reply:
Hi, thanks for the suggestion. I added that part

Reply:
Looking good, thanks.